myapp/config/locales/en.yml

```diff
@@ -53,12 +53,13 @@ en:
       not started: "Not Started"
       in progress: "In Progress"
       done: "Done"
-
+  
```

Comment:
Is this difference intended? if no, you can revert. 👀

Reply:
Reverted it back.